clustering.py

```diff
@@ -0,0 +1,36 @@
+# import libraries
+import pandas as pd
+import matplotlib.pyplot as plt
+from sklearn.cluster import KMeans
+from scipy import stats
+import seaborn as sns
+
+
+def make_map():
+    data = pd.read_csv('example/out.tips')
+
+    data = data[["ag1", "ag2"]]
+    # data = stats.zscore(data)
+
+    kmeans = KMeans(n_clusters=11, n_init=40, max_iter=100, random_state=0).fit(data)
```

Comment:
The arguments for `KMeans()` could also be specified in this argument parser as well. It would be smart to leave some defaults though for `n_init`, `max_iter` and `random_state`.